frontend/src/components/common/comments/CommentPanel.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Spin, Comment, List, Tooltip } from "antd";
+
+import CommentEditor from "./CommentEditor";
+
+const propTypes = {
+  submitting: PropTypes.bool.isRequired,
+  loading: PropTypes.bool.isRequired,
+  renderAdd: PropTypes.bool,
+  comments: PropTypes.arrayOf(PropTypes.any).isRequired,
+  onSubmit: PropTypes.func.isRequired,
+  onChange: PropTypes.func,
+};
+
+const defaultProps = {
+  renderAdd: true,
+  onChange: () => {},
+};
+
+const CommentPanel = (props) => {
+  if (!props.loading)
```

Comment:
It looks like this is just a ternary disguised as an if block? Would this work for you here?
```javascript
!props.loading ? (
    <React.Fragment>
        // stuff
    </React.Fragment>
) : (
    <Spin />
);
```

This is how our other components handle this sort of simple conditional rendering regarding loading state

Reply:
Yeah, that would work fine - I'll make the change.